src/registrar/views/application.py

```diff
@@ -0,0 +1,421 @@
+import logging
+
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import Http404
+from django.shortcuts import redirect, render
+from django.urls import resolve, reverse
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import TemplateView
+
+from registrar.forms import application_wizard as forms
+from registrar.models import DomainApplication
+from registrar.utility import StrEnum
+from registrar.views.utility import StepsHelper
+
+logger = logging.getLogger(__name__)
+
+
+class Step(StrEnum):
```

Comment:
**suggestion (non-blocking)**: Because the `StrEnum` class is non-standard, how about a comment here that the order in which these constants are defined will be the order that the steps appear? At least I think that is the way that it will work?

Reply:
Comment added.